lib/flapjack/data/entity_check.rb

```diff
@@ -606,6 +606,8 @@ def update_state(new_state, options = {})
         details = options[:details]
         perfdata = options[:perfdata]
         count = options[:count]
+        initial_failure_delay = options[:initial_failure_delay]
+        repeat_failure_delay = options[:repeat_failure_delay]
 
```

Comment:
These local variables are shadowing the instance method name; I'll change that to make it explicit that later usages aren't meant to call the method.